### PR TITLE
Don't load zamboni/unicode.js dynamically

### LINKED ISSUE #1349: Don't load zamboni/unicode.js dynamically using jquery
Loading a script dynamically using jquery will make use of `eval`, which we forbid using CSP.

### PR DESCRIPTION
**Pull request by magopian**
Fixes #1349

### CODE REVIEW

**Comment by muffinresearch** [comment]
Won't this mean the unicode_loaded event will fire before the collections is bound? 

See https://github.com/mozilla/olympia/blob/61e5bdfa2300192ca65edb0a3cc8228d4bbd3183/static/js/zamboni/collections.js#L554

**Comment by magopian** [comment]
Well, obviously it needs to be the other way round. Duh.

File: lib/settings_base.py [hunk 1]
```diff
@@ -633,6 +633,10 @@ def JINJA_CONFIG():
             # Collections
             'js/zamboni/collections.js',
 
+            # Unicode: needs to be loaded before collections.js which uses an
```
Comment (muffinresearch) [inline lib/settings_base.py]: s/before/after

Comment (muffinresearch) [inline lib/settings_base.py]: Also the 2nd line doesn't appear to make sense. Could just say "which listens to an event fired in this file" or something.

File: lib/settings_base.py [hunk 2]
```diff
@@ -724,6 +728,10 @@ def JINJA_CONFIG():
             'js/zamboni/collections.js',
             'js/impala/collections.js',
 
+            # Unicode: needs to be loaded after collections.js which uses an
+            # event trigger from unicode.js.
```
Comment (muffinresearch) [inline lib/settings_base.py]: As above

**Comment by muffinresearch** [comment]
r+wc

**Pull request merged**